test.js

```diff
@@ -0,0 +1,34 @@
+const assert = require('assert');

```

Comment:
u should have 1 seperate test file for each function. At least u should write `desrcibe` for a separate function, but not for all of them.
Here is the expample:
```suggestion
const func1 = function(){};
const func2 = function(){};
const func3 = function(){};

describe('func1', () => {
  it('does smth', () => {
    assert.deepEqual(sumOfOther([]), []);
  });
});

describe('func2', () => {
  it('does smth', () => {
    assert.deepEqual(sumOfOther([]), []);
  });
});

describe('func3', () => {
  it('does smth', () => {
    assert.deepEqual(sumOfOther([]), []);
  });
});

```